src/seals/testing/envs.py

```diff
@@ -148,10 +154,9 @@ def test_seed(env: gym.Env, env_name: str, deterministic_envs: Iterable[str]) ->
     """
     env.action_space.seed(0)
     actions = [env.action_space.sample() for _ in range(10)]
-
     # With the same seed, should always get the same result
     seeds = env.seed(42)
-    assert isinstance(seeds, list)
+    assert isinstance(seeds, collections.abc.Sequence)
     assert len(seeds) > 0
```

Comment:
Why check this here and not on `rollout_b`?

Reply:
I guess because we're checking the output type of `env.seed`, which presumably is guaranteed to be the same both times, barring something going horribly wrong?